src/user/components/change_password_form.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+const form_fields = [ 'new_password', 'confirm_password' ]
+
+export default ({fields, handleSubmit}) =>
+  <form onSubmit={handleSubmit}>
+    <div className='change-password-inputs'>
+      {form_fields.map(input_maker(fields))}
+    </div>
+    <button className='change-password-button' type='submit'>Submit</button>
+  </form>
+
+
+const input_maker = fields => field =>
```

Comment:
Further props are passed through, so `type='password'` should still work. And you can just stick `mode='edit'` on the field hardcoded.

I see there are some advantages to having the custom field, but I think they're outweighed by the disadvantages / advantages of reusing